pipelines/bazel-release.yml

```diff
@@ -66,7 +66,19 @@ steps:
       mkdir output
       cp bazel-bin/src/bazel output/bazel
 
-      output/bazel build -c opt --stamp --sandbox_tmpfs_path=/tmp --embed_label "\${release_name}" --workspace_status_command=scripts/ci/build_status_command.sh src/bazel scripts/packages/with-jdk/install.sh scripts/packages/debian/bazel-debian.deb scripts/packages/debian/bazel.dsc scripts/packages/debian/bazel.tar.gz bazel-distfile.zip
+      output/bazel build \
+          -c opt \
+          --stamp \
+          --sandbox_tmpfs_path=/tmp \
+          --embed_label "\${release_name}" \
+          --workspace_status_command=scripts/ci/build_status_command.sh \
+          --host_force_python=PY2 \
```

Comment:
Suggest adding a comment saying why we need this / what part breaks.

Reply:
Done!